project/dexter-server/public/defect/defectCtrl.js

```diff
@@ -453,6 +406,60 @@ defectApp.controller('DefectCtrl', function ($scope, $http, $sce, $location, $an
         }).catch(console.error);
     };
 
+    function toggleShowFileTreeBtn(isShowState) {
+        $scope.isFileTreeHidden = isShowState;
+        $scope.isFileTreeBtnTitleHidden = ((isShowState === true) ? 'Show File Tree' : 'Hide File Tree');
+    }
+
+    function checkLogin() {
+        $http.get("/api/v1/accounts/checkLogin", {
+        }).then(function (results) {
+            if (results.data.userId) {
+                $scope.isHideLoginBtnTitle = results.data.userId + "/ logout";
+            }
+            else {
+                $scope.isHideLoginBtnTitle = 'Login';
+            }
+        }, function (results) {
+            $log.error(results);
+        });
+    }
+
+    function initLocalStorage() {
+        window.localStorage['pageSize'] = (window.localStorage['pageSize']) || 500;
+        window.localStorage['currentPage'] = parseInt(window.localStorage['currentPage']) || 1;
+    }
+
+    function getProjectName() {
+        $http.get('/api/v1/projectName', {}).then(function (result) {
+            if (result && result.data) {
+                var html = 'Defect : ' + result.data.projectName;
+                $scope.projectName = result.data.projectName;
+                $('#indexTitle').html(html);
+            }
+        }, function (results) {
+            $log.error(results);
+        });
+    }
+
+    function moveTopOfPage() {
+        $location.hash('bookmark');
+        $anchorScroll();
+    }
+
+    function removeLocalStorageOfResources() {
```

Comment:
next time, please change the function name.
I can not imagine modulePath, fileName, and selectTree from 'Resources'.
If this function removes all local storage, you can use 'removeAllLocalStorage'.